smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SymbolVisitor.kt

```diff
@@ -157,7 +157,7 @@ class SymbolVisitor(private val model: Model, swiftSettings: SwiftSettings) :
         // add a reference to each member symbol
         addDeclareMemberReferences(builder, shape.allMembers.values)
 
-        if (shape.hasTrait<NestedTrait>() && service != null) {
+        if (shape.hasTrait<NestedTrait>() && service != null && !shape.hasTrait<ErrorTrait>()) {
```

Comment:
Is there any way to write a unit test for this?

Reply:
will do!